pir/fast_pir/fast_pir_client.hpp

```diff
@@ -14,13 +14,21 @@
 using std::array;
 using std::bitset;
 
+#define DUMMY_INDEX 1'000'000
```

Comment:
why is dummy_index not 0 anymore? is there an attack i'm missing

Reply:
basically, if you are sending to both the friend the index 0 and dummy, there is some problem. This causes overwrite bugs. — Shengtong.